packages/web-components/fast-components/src/search/search.styles.ts

```diff
@@ -1,226 +1,111 @@
-        border-color: ${accentFillHover};
-    }
-
-    :host(:active:not([disabled])) .root {
-        background: ${neutralFillInputHover};
-        border-color: ${accentFillActive};
-    }
-
-    :host(:focus-within:not([disabled])) .root {
-        border-color: ${focusStrokeOuter};
-        box-shadow: 0 0 0 1px ${focusStrokeOuter} inset;
-    }
-
-    .clear-button__hidden {
-        opacity: 0;
-    }
-
-    :host(:hover:not([disabled], [readOnly])) .clear-button,
-    :host(:active:not([disabled], [readOnly])) .clear-button,
-    :host(:focus-within:not([disabled], [readOnly])) .clear-button {
-        opacity: 1;
-    }
-
-    :host(:hover:not([disabled], [readOnly])) .clear-button__hidden,
-    :host(:active:not([disabled], [readOnly])) .clear-button__hidden,
-    :host(:focus-within:not([disabled], [readOnly])) .clear-button__hidden {
-        opacity: 0;
-    }
-
-    :host([appearance="filled"]) .root {
-        background: ${fillColor};
-    }
-
-    :host([appearance="filled"]:hover:not([disabled])) .root {
-        background: ${neutralFillHover};
-    }
-
-    :host([disabled]) .label,
-    :host([readonly]) .label,
-    :host([readonly]) .control,
-    :host([disabled]) .control {
-        cursor: ${disabledCursor};
-    }
-
-    :host([disabled]) {
-        opacity: ${disabledOpacity};
-    }
+        ${inputFilledStyles(context, definition, ".root")}
+    `.withBehaviors(
+        forcedColorsStylesheetBehavior(
+            css`
+                ${inputFilledForcedColorStyles(context, definition, ".root")}
+            `
+        )
+    );
 
-    :host([disabled]) .control {
-        border-color: ${neutralStrokeRest};
-    }
+/**
+ * Styles for Search
+ * @public
+ */
+export const searchStyles = (context, definition) =>
+    css`
+        ${display("inline-block")}
+        ${inputStyles(context, definition, ".root")}
+        ${inputStateStyles(context, definition, ".root")}
+        .root {
+            display: flex;
+            flex-direction: row;
+        }
+
+        .control {
+            -webkit-appearance: none;
+            color: inherit;
+            background: transparent;
+            border: 0;
+            height: calc(100% - 4px);
+            margin-top: auto;
+            margin-bottom: auto;
+            padding: 0 calc(${designUnit} * 2px + 1px);
+            font-family: inherit;
+            font-size: inherit;
+            line-height: inherit;
+        }
+
+        .control::-webkit-search-cancel-button {
+            -webkit-appearance: none;
+        }
+
+        .start,
+        .end {
+            display: flex;
+            margin: 1px;
+        }
+
+        ::slotted([slot="end"]) {
+            height: 100%
+        }
+
+        ::slotted(svg) {
+            margin-inline-end: 11px;
+            margin-inline-start: 11px;
+            margin-top: auto;
+            margin-bottom: auto;
+        }
+
+        .clear-button {
+            position: absolute;
```

Comment:
Please refer too fluent search box this should no longer need to be absolutely positioned. Change this to `inlne-flex` and make sure the `.input-wrapper` class is set to `flex`

Reply:
set to inline-flex and added input-wrapper selector